src/scripts/seqbox_utils.py

```diff
@@ -1169,10 +1203,11 @@ def check_extraction_fields(extraction_info):
     if extraction_info['sample_identifier'].strip() == '':
         print(f'sample_identifier column should not be empty. it is for \n{extraction_info}\nExiting.')
         sys.exit(1)
-    if extraction_info['date_extracted'].strip() == '':
+    
+    if str(extraction_info['date_extracted']) == 'NaT':
```

Comment:
What is 'NaT'?

Reply:
add answer as a comment

Reply:
convert NaTs to None in the read_excel, like the NaNs

Reply:
> What is 'NaT'?

Not a Time object. Pandas use it to refer to missing datetime values